test/run_and_collect_test.dart

```diff
@@ -40,42 +38,30 @@ void main() {
       expect(sampleCoverageData['hits'], isNotEmpty);
     }
 
-    final hitMap = await createHitmap(
-      coverage,
-      checkIgnoredLines: true,
-    );
-    expect(hitMap, contains(_sampleAppFileUri));
+    final hitMap = await createHitmap(coverage, checkIgnoredLines: true);
+    expect(hitMap, isNot(contains(_sampleAppFileUri)));
 
     final actualHits = hitMap[_isolateLibFileUri];
     final expectedHits = {
+      11: 1,
       12: 1,
       13: 1,
       15: 0,
+      28: 1,
+      29: 1,
+      31: 1,
+      32: 3,
+      46: 1,
+      47: 1,
+      18: 1,
       19: 1,
       20: 1,
       22: 0,
-      29: 1,
-      31: 1,
-      32: 2,
       33: 1,
       34: 3,
-      35: 1,
-      46: 1,
-      47: 1,
+      35: 1
     };
-    // Dart VMs prior to 2.0.0-dev.5.0 contain a bug that emits coverage on the
-    // closing brace of async function blocks.
-    // See: https://github.com/dart-lang/coverage/issues/196
-    if (Platform.version.startsWith('1.')) {
-      expectedHits[23] = 0;
-    } else {
-      // Dart VMs version 2.0.0-dev.6.0 mark the opening brace of a function as
-      // coverable.
-      expectedHits[11] = 1;
-      expectedHits[18] = 1;
-      expectedHits[28] = 1;
-      expectedHits[32] = 3;
-    }
```

Comment:
Haha fair enough; it's about time. We should probably add a note to the changelog about this removal of support.